Ui/Component/Listing/Column/SyncedCreditmemo.php

```diff
@@ -57,9 +58,14 @@ public function prepareDataSource(array $dataSource)
     {
         if (isset($dataSource['data']['items'])) {
             foreach ($dataSource['data']['items'] as & $item) {
-                $creditmemo = $this->creditmemoRepository->get($item['entity_id']);
                 $creditmemoSyncDate = '';
 
+                try {
+                    $creditmemo = $this->creditmemoRepository->get($item['entity_id']);
+                } catch( NoSuchEntityException $e) {
+                    $this->logger->log($e->getMessage() . ', entity id: ' . $item['entity_id']);
+                }
+
                 if ($creditmemo->getTjSalestaxSyncDate()) {
```

Comment:
If the exception is caught, `$creditmemo` may not be set. Should we check to see if `$creditmemo` exists for this conditional to avoid an additional error?

```php
if ($creditmemo && $creditmemo->getTjSalestaxSyncDate())
```